### ISSUE TITLE
Widget Not Added to List

### ISSUE THREAD

**Issue by Btibert3** [biolab/orange3-example-addon#14]
I recently posted this to the Discord channel.  In short, I am interested in building my own widgets, but I can't seem to figure out how to get the widgets to display after installing.

In order to work backwards, I cloned this repo, installed, and the same issue; the widgets are not appearing in the canvas.

Also posted here:  https://discord.com/channels/633376992607076354/822470786346516501/949118129848610816

In short, I don't believe the current example works as designed.

**Comment by markotoplak** [comment]
Thanks for reporting this. I confirm your issue. The last commit, 54a091b5, broke the package because some imports are missing. Just a minute, I'll add them.

**Comment by markotoplak** [comment]
@Btibert3, I think we fixed this one. Could you try again? Thanks!

**Comment by Btibert3** [comment]
Thanks for the quick fix.  I will be able to take a look in the morning;
this is exciting thank you!

**Comment by Btibert3** [comment]
Verified.  Thank you very much.
